shared/shared/postgresql_backend/pg_connection_handler.py

```diff
@@ -91,7 +91,7 @@ def __init__(self, shutdown_service: ShutdownService):
         shutdown_service.register(self)
         self._storage = threading.local()
 
-        min_conn = int(self.environment.try_get("DATASTORE_MIN_CONNECTIONS") or 1)
+        min_conn = int(self.environment.try_get("DATASTORE_MIN_CONNECTIONS") or 0)
```

Comment:
I had to change the minimum here to 0, since with 1 it immediatly tries to connect to the database, which fails in the github action for the productive container in the backend since it is started without a database there. But I think this is fine; a connection is established anyway as soon as it is needed.